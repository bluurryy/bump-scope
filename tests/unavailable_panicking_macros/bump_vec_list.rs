use bump_scope::{Bump, BumpVec, bump_vec};

pub fn one_two_three<'a>(bump: &'a Bump, two: i32) -> BumpVec<i32, &'a Bump> {
    bump_vec![in bump; 1, two, 3]
}

fn main() {}
