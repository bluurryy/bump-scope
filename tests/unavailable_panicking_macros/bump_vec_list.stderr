error: the potentially panicking api of `bump_vec!` is not available
       help: enable `bump-scope`'s "panic-on-alloc" feature or use `try`:
       `bump_vec![try in bump; 1, two, 3]`
 --> tests/unavailable_panicking_macros/bump_vec_list.rs:4:5
  |
4 |     bump_vec![in bump; 1, two, 3]
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `$crate::__bump_vec_panic_on_alloc` which comes from the expansion of the macro `bump_vec` (in Nightly builds, run with -Z macro-backtrace for more info)
